app/views/admin/index.html.erb

```diff
@@ -0,0 +1,42 @@
+
+  <div class="d-flex flex-column justify-content-start">
```

Comment:
Gorgeous landing page. Changing justify-content-start to justify-content-between will center the two elements instead of being left justified.